src/api-docs/category.docs.js

```diff
@@ -0,0 +1,48 @@
+
+const category = {
```

Comment:
Same as here I suggest renaming this variable to something like postCategory or you can choose any other clear and descriptive name.